src/components/DonutChart.js

```diff
@@ -71,7 +71,7 @@ class DonutChart extends React.Component {
           return (
             <g
               key={i}
-              onClick={this.props.onClick.bind(this, i)}
+              onClick={this._handleClick.bind(this, i)}
```

Comment:
Yeah... that might be good to have. Added and pushed that up. Should be ready for review.